tests/unit/parsers/test.manifestjson.js

```diff
@@ -1456,12 +1456,19 @@ describe('ManifestJSONParser', () => {
         // forseeable future. See http://bit.ly/2wG6LP0 for more details-
         "script-src 'self' 'unsafe-inline';",
 
+        // 'wasm-unsafe-eval' is permitted, despite the unsafe-eval substring.
+        "script-src 'self' 'wasm-unsafe-eval'",
+
         // 'default-src' is insecure, but the limiting 'script-src' prevents
         // remote script injection
         "default-src *; script-src 'self'",
         "default-src https:; script-src 'self'",
         "default-src example.com; script-src 'self'",
         "default-src http://remote.com/; script-src 'self'",
+        // This should be accepted because script-src overrides default-src,
+        // but because unsafe-eval is unconditionally rejected despite the
+        // script-src override, the linter complains anyway:
+        // "script-src 'wasm-unsafe-eval'; default-src 'unsafe-eval'",
```

Comment:
Should we fix that somehow?

Reply:
I have considered doing so, but decided not to since it's not really a big deal to block its use from `default-src`.

`'unsafe-eval'` is only meaningful in `script-src` and `style-src`:
- https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/script-src#unsafe_eval_expressions
- https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/style-src#unsafe_style_expressions

... and `style-src` is not validated here. Extensions wanting to use the `unsafe-eval` part of the CSP should just use `style-src` instead of `default-src`, for clarity.

Reply:
ok, can you maybe rework the comment a bit?